meson.build

```diff
@@ -520,6 +520,15 @@ motd_dir = 'motd.d'
 conf.set_quoted('MOTD_FILE', motd_file)
 conf.set_quoted('MOTD_DIR', motd_dir)
 
+if host_machine.system() == 'windows'
+  if host_machine.cpu() != 'x86_64'
+    wixl_arch = 'x86'
+  else
+    wixl_arch = 'x64'
```

Comment:
<cough> arm </cough>

Reply:
I'm guessing it's `host_machine.cpu() of "aarch64" to "arm64"`?